src/applications/facility-locator/components/AppointmentInfo.jsx

```diff
@@ -71,72 +66,72 @@ export default class AppointmentInfo extends Component {
       return null;
     };
 
-    const renderSpecialtyTimes = (existing = false) => {
-      const specialtyKeys = healthAccessAttrs && Object.keys(healthAccessAttrs);
-      pull(specialtyKeys, 'primaryCare', 'effectiveDate');
-      specialtyKeys.sort();
-
-      if (specialtyKeys && specialtyKeys.length === 0) {
-        return null;
-      }
-
-      const firstThree = specialtyKeys.slice(0, 3);
-      const lastToEnd = specialtyKeys.slice(3);
-      let showHideKey;
-
-      if (existing) {
-        showHideKey = 'newPatientTimesExpanded';
+    const renderPrimaryCare = (accessAttrs, category) => {
+      if (Array.isArray(accessAttrs)) {
+        // V1
+        const value = accessAttrs.find(
+          k => k.service === 'PrimaryCare' && k[category],
+        );
+        return renderStat('Primary Care', value[category]);
       } else {
-        showHideKey = 'existingPatientTimesExpanded';
+        // V0 legacy structure TODO: remove when fully migrated to V1
+        const value = get(accessAttrs, ['primaryCare', category]);
+        if (value || value === 0) {
```

Comment:
I'm not seeing why we need this `if` block here, since we aren't using `value` inside the block.

Reply:
Ah - I think I see now - we only want to call `renderStat` if there is a value. 

We may want to do `Number.isInteger(value)` instead?